moveit_setup_assistant/templates/moveit_config_pkg_template/launch/planning_pipeline.launch.xml

```diff
@@ -4,7 +4,8 @@
        It is assumed that all planning pipelines are named XXX_planning_pipeline.launch  -->
 
   <arg name="pipeline" default="ompl" />
+  <arg name="pipeline_pkg" default="[GENERATED_PACKAGE_NAME]" />
 
-  <include file="$(find [GENERATED_PACKAGE_NAME])/launch/$(arg pipeline)_planning_pipeline.launch.xml" />
+  <include file="$(find ${[GENERATED_PACKAGE_NAME]})/launch/$(arg pipeline)_planning_pipeline.launch.xml" />
```

Comment:
I think, `[GENERATED_PACKAGE_NAME]` should be replaced by pipeline_pkg here to be useful.
However, I agree with @v4hn to directly specify the xml file.